examples/python/gpio_advanced.py

```diff
@@ -21,34 +21,39 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE
+#
+# Example Usage: Reads GPIO 6 through interrupt
 
 import mraa
 import time
 import sys
 
 class Counter:
-  count = 0
+    count = 0
 
 c = Counter()
 
 # inside a python interrupt you cannot use 'basic' types so you'll need to use
 # objects
-def test(gpio):
-  print("pin " + repr(gpio.getPin(True)) + " = " + repr(gpio.read()))
-  c.count+=1
+def isr_routine(gpio):
+    print("pin " + repr(gpio.getPin(True)) + " = " + repr(gpio.read()))
+    c.count+=1
```

Comment:
While we are at it, this needs spaces around `+=`

Reply:
Okay. Will add spaces around `=` in all examples